pkg/users/account-details.js

```diff
@@ -145,13 +155,14 @@
                                     console.log("Account locked state doesn't match desired value, trying again.");
                                     // only retry once to avoid uncontrolled recursion
                                     change_locked(value, true);
-                                } else
-                                    set_edited_locked(null);
+                                }
                             });
                 })
                 .catch(error => {
-                    set_edited_locked(null);
                     show_unexpected_error(error);
+                })
+                .finally(() => {
+                    setDisableLockedEdit(false);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-5e987a2f-20250813-131607-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)